lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -105,6 +105,13 @@ def self.filters_schema
                  has_default: true,
                  visibility: false
 
+          schema :timeline_zoom_level,
+                 type: 'String',
+                 required: false,
+                 writable: true,
+                 has_default: true,
+                 visibility: false
```

Comment:
Thought about this too while writing the comment but as I believe it to be rather simple and I do see the benefit in communicating the allowed values, I believe we should do it.

You should be able to use `schema_with_allowed_collection` and communicate the values as string objects.

I could also do it if you like to, after I finished my current topic.